Yen/learnNavigation/app/src/main/java/com/example/learnnavigation/view/fragment/navigation/FragmentHost.kt

```diff
@@ -0,0 +1,69 @@
+package com.example.learnnavigation.view.fragment.navigation
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import com.example.learnnavigation.R
+import com.example.learnnavigation.adapter.ViewPage2Adapter
+import com.example.learnnavigation.databinding.FragmentHostBinding
+import com.example.learnnavigation.view.fragment.FragmentAnimation
+import com.example.learnnavigation.view.fragment.FragmentRing
+import com.example.learnnavigation.view.fragment.FragmentBackgroundImage
+import com.example.learnnavigation.view.fragment.FragmentScreen
+
+class FragmentHost : Fragment(R.layout.fragment_host) {
+
+    private lateinit var binding: FragmentHostBinding
+    private lateinit var adapter: ViewPage2Adapter
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentHostBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupViewPager()
+        setupBottomNavigationView()
+    }
+
+    private fun setupViewPager() {
+        val fragmentList = arrayListOf(
```

Comment:
list này chưa fragment và idMenu ví dụ listOf(Pair(FragmentHome(),R.id.home) -> fun getPositionFragment(idMenu: Int) = listFragment.indexOf{it.second == idMenu}